modules/adding-tab-pods-page.adoc

```diff
@@ -1,14 +1,48 @@
 // Module included in the following assemblies:
 //
-// * web_console/dynamic-plug-ins.adoc
+// * web_console/dynamic-plugin-example.adoc
 
 :_content-type: PROCEDURE
 [id="adding-tab-to-pods-page_{context}"]
 = Adding a tab to the pods page
-The following procedure adds a tab to the *Pod Details* page as an example extension to your plug-in.
+
+[NOTE]
+====
+The {product-title} web console runs in a container connected to the cluster you have logged into. See "Dynamic plug-in development" for information to test the plug-in before creating your own.
+====
 
 .Procedure
 
+. Visit the link:https://github.com/openshift/console-plugin-template[`console-plugin-template`] repository containing a template for creating plug-ins in a new tab.
+
+. Select *Use this Template* to create a GitHub repository.
```

Comment:
I didn't see the button right off. Maybe _Select the *Use this Template* button.._ ??

Reply:
Do you want to specify which option to use with that button? I assume _Create a new repository_, but I wouldn't be certain.